"""
    The exception we throw for all problems in Evaluation
"""

class EvaluationException(Exception):
    pass